README.rst

```diff
@@ -228,7 +232,9 @@ Then use it::
 
 Further, you can auto-generate configuration documentation by including the
 ``everett.sphinxext`` Sphinx extension and using the ``autocomponent``
-directive::
+directive:
+
+.. code:: rst
```

Comment:
`.. code::` isn't in reST. `twine check` seems fine with this and I'm not sure why. reST defines `code-block` in the way you're using here.

So this should be:

```
.. code-block:: rst

   .. autocomponent:: path.to.AppConfig
```